ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -0,0 +1,68 @@
+
+if have_tensorflow
+  tensor_filter_tfcore_lib_shared = shared_library('nnstreamer_filter_tensorflow',
+      [
+        'tensor_filter_tensorflow_core.cc',
+        'tensor_filter_tensorflow.c'
+      ],
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: [tf_dep, protobuf_dep, glib_dep, gst_dep, orc_dep, nnstreamer_dep],
+    include_directories: nnstreamer_inc,
+    install: true,
+    install_dir: filter_subplugin_install_dir
+  )
+
+  tensor_filter_tfcore_lib_static = static_library('nnstreamer_filter_tensorflow',
+      [
+        'tensor_filter_tensorflow_core.cc',
+        'tensor_filter_tensorflow.c'
+      ],
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: [tf_dep, protobuf_dep, glib_dep, gst_dep, orc_dep, nnstreamer_dep],
```

Comment:
Later work, 
1. About the dependency nnstreamer_dep, now we declared nnstreamer_dep with nnstreamer base dependencies (glib, gst, gst-video/audio, orc, ..). Is it necessary to add this base dependencies?
2. Do we need to build static library for sub-plugins?

Reply:
1: Nope. I don't think so.

2: for Android project, we have the needs, releasing a whole big statically built binary containing the whole pipeline, gst, gst-plugins, nns, subplugins, nnfws, and custom filters.